### PR TITLE
Feat/prisma psql

### PR DESCRIPTION
**Pull request by jasnwng**

### CODE REVIEW

**Review by SunnyAnter — DISMISSED** [review]

File: .env.local.template [hunk 1]
```diff
@@ -1,3 +1,6 @@
 NODE_ENV=development
 MONGO_URI='mongodb://<user>:<pass>@<host>:<port>/<database>?<connection options>'
-NEXT_PUBLIC_MAPBOX_TOKEN=[ENTER_MAP_API_KEY]
\ No newline at end of file
+NEXT_PUBLIC_MAPBOX_TOKEN=[ENTER_MAP_API_KEY]
+
+
```
Comment (TobyDS) [inline .env.local.template:5]: Remove empty space

File: app/api/db/route.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        'Thursday: 6:30 AM – 10:00 PM',
+        'Friday: 6:30 AM – 10:00 PM',
+        'Saturday: 6:30 AM – 4:00 PM',
+        'Sunday: Closed',
+      ],
+      summary: 'A chalkboard menu of Italian classic dishes & British fare in a relaxed counter-service setting.',
+      image_url: 'null',
+      latitude: 51.4949702,
+      longitude: -0.1277006,
+    },
```
Comment (TobyDS) [inline app/api/db/route.ts:32]: Move mock data to a separate file, ideally in a `/data` folder

File: app/dashboard/db/page.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+async function db () {
+  const res = await fetch('http://localhost:3000/api/db');
```
Comment (TobyDS) [inline app/dashboard/db/page.tsx:4]: Make the base URL read from an environment variable

File: app/lib/prisma.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { PrismaClient } from '@prisma/client';
+// const prisma = new PrismaClient();
```
Comment (TobyDS) [inline app/lib/prisma.ts:2]: Remove dead code

File: app/api/route.ts
Comment (TobyDS) [inline app/api/route.ts:1]: Empty file should not be included in pull request

File: prisma/schema.prisma [hunk 5]
```diff
@@ -0,0 +1,38 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Try Prisma Accelerate: https://pris.ly/cli/accelerate-init
```
Comment (TobyDS) [inline prisma/schema.prisma:5]: Remove

**Review by TobyDS — CHANGES_REQUESTED** [review]

**Review by TobyDS — DISMISSED** [review]

**Review by TobyDS — APPROVED** [review]

**Review by SunnyAnter — APPROVED** [review]

**Pull request merged**